perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ConsentsManagerMethod.java

```diff
@@ -81,5 +83,27 @@ public ConsentHub call(ApiCaller ac, Deserializer parms) throws PerunException {
 			return ac.getConsentsManager().updateConsentHub(ac.getSession(),
 				parms.read("consentHub", ConsentHub.class));
 		}
+	},
+
+	/*#
+	 * Returns consent with changed status.
+	 *
+	 * @param consent int Consent <code>id</code>
+	 * @param status String UNSIGNED | GRANTED | REVOKED
+	 * @param user int User <code>id</code>
+	 *
+	 * @throws ConsentNotExistsException
+	 * @throws PrivilegeException
+	 * @throws UserNotExistsException
+	 * @throws InvalidConsentStatus
```

Comment:
```suggestion
	 * @throw InvalidConsentStatus
```
I think it should be like this here in RPC (I think it's because of parsing, I am not sure if _throws_ would be incorrect, tho). Also, `PrivilegeException` is added automatically to every method in RPC, so we should omit it. 

Please, add a brief description of when these exceptions happen.